listener/upload.go

```diff
@@ -25,6 +25,8 @@ const (
 	ErrorProcessUpload     = "unable to process upload"
 	UploadSuccessNoEval    = "upload event handled successfully, no eval required"
 	UploadSuccess          = "upload event handled successfully"
+
+	DeletionThreshold = 4 * time.Hour
```

Comment:
would be better to create env var for that with 4h default value

Reply:
Thanks, fixed